.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: CI
+
+on: ["push", "pull_request"]
```

Comment:
I think `pull_request` is not necessary here if the action is run on each push in all branches, it will just run the check on the same commit twice (after push and on pull request).